components/aws/sagemaker/tests/integration_tests/resources/config/assume-role-processing/config.yaml

```diff
@@ -0,0 +1,47 @@
+PipelineDefinition: resources/definition/processing_pipeline.py
+TestName: kmeans-algo-mnist-processing
```

Comment:
change name of the test, also config files are created but not added to the tests